resources/user-agents/browsers/whale/whale.json

```diff
@@ -32,6 +32,14 @@
             "Win7_64", "Win7_32",
             "Windows"
           ]
+        },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Chrome/* Whale/* Safari/*",
```

Comment:
Recently released for macOS (as a beta, but I didn't want to have to worry about removing the beta tag later):

http://whale.naver.com/en/download